packages/app/src/ui/pages/DownloadBackup/DownloadBackup.tsx

```diff
@@ -33,9 +33,9 @@ const DownloadBackup = (): JSX.Element => {
             <Typography variant="body1">Backup contains encrypted data:</Typography>
 
             <List dense>
-              <ListItem>- Your current password</ListItem>
+              <ListItem>- Your current password and mnemonic</ListItem>
 
-              <ListItem>- Your identities</ListItem>
+              <ListItem>- Your identities and accounts</ListItem>
```

Comment:
What is the difference between `identities` and `accounts`?

Reply:
With account user can sign the message and create an identity

Reply:
That might confuse users between `identities` and `accounts`. Could we maybe say `Ethereum accounts`?

Reply:
Yup that makes sense as well. :ok_hand: